analysis_driver/pipelines/__init__.py

```diff
@@ -19,15 +19,23 @@ def pipeline(dataset):
     elif isinstance(dataset, SampleDataset):
         species = clarity.get_species_from_sample(dataset.name)
         genome_version = clarity.get_sample(dataset.name).udf.get('Genome Version')
+        bcbio_genome = cfg['genome']
         analysis_type = clarity.get_sample(dataset.name).udf.get('Analysis Type')
         if species is None:
             raise PipelineError('No species information found in the LIMS for ' + dataset.name)
-        elif species == 'Homo sapiens':
-            return bcbio_var_calling_pipeline(dataset, genome_version, analysis_type)
-        elif clarity.get_sample(dataset.name).udf.get('Analysis Type') == 'Variant Calling':
-            return var_calling_pipeline(dataset, species)
+        if genome_version is None:
+            genome_version = cfg.query('species', species, 'default')
+            reference = cfg.query('genomes', genome_version, 'default')
+        else:
+            reference = cfg.query('genomes', genome_version, 'fasta')
+        if species == 'Homo sapiens':
+            return bcbio_var_calling_pipeline(dataset, bcbio_genome, analysis_type)
```

Comment:
bcbio's `genome_version` won't be working anymore as it will force to use `bcbio_genome`